docs/release-notes/rockcraft-1-14.rst

```diff
@@ -0,0 +1,160 @@
+.. _release-1.14:
+
+..
+    rockcraft
+    docs: Incorporate UX feedback into 12-factor tutorials @erinecon (#918)
+    docs: add CONTRIBUTING.md @medubelko (#925)
+    fix: add missing migrate and migrate.sh @javierdelapuente (#932)
+    feat: add entrypoint-command @alesancor1 (#939)
+
+     name = "craft-application"
+    -version = "5.4.0"
+    +version = "5.7.1"
+
+    feat(TestCommand): pack and test all packable platforms by @lengau in #807
+
+     name = "craft-archives"
+    -version = "2.1.0"
+    +version = "2.2.0"
+
+    fix: retry key receiving on port TCP/80 if using default keyserver by @upils in #189
+
+     name = "craft-cli"
+    -version = "3.0.0"
+    +version = "3.1.2"
+
+     name = "craft-grammar"
+    -version = "2.0.3"
+    +version = "2.2.0"
+
+     name = "craft-parts"
+    -version = "2.16.0"
+    +version = "2.20.1"
+
+    docs: add maven-use reference page by @bepri in #1179
+    feat(jlink): introduce add_modules parameter by @vpa1977 in #1169
+    fix(overlay): detect conflicts between overlay and install by @tigarmo in #1190
+
+Rockcraft 1.14 release notes
+============================
+
+29 August 2025
+
+Learn about the new features, changes, and fixes introduced in Rockcraft 1.14.
+For information about the Rockcraft release cycle, see the
+:ref:`release_policy_and_schedule`.
+
+
+Requirements and compatibility
+------------------------------
+
+To run Rockcraft, a system requires the following minimum hardware and
+installed software. These requirements apply to local hosts as well as VMs and
+container hosts.
+
+
+Minimum hardware requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- AMD64, ARM64, ARMv7-M, RISC-V 64-bit, PowerPC 64-bit little-endian, or S390x
+  processor
+- 2GB RAM
+- 10GB available storage space
+- Internet access for remote software sources and the Snap Store
+
+
+Platform requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+  :header-rows: 1
+  :widths: 1 3 3
+
+  * - Platform
+    - Version
+    - Software requirements
+  * - GNU/Linux
+    - Popular distributions that ship with systemd and are `compatible with
+      snapd <https://snapcraft.io/docs/installing-snapd>`_
+    - systemd
+
+
+What's new
+----------
+
+Rockcraft 1.14 brings the following features, integrations, and improvements.
+
+New ``entrypoint-command`` project key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This key defines the rock's OCI entrypoint, bypassing the Pebble-powered entrypoint
+defined by Rockcraft. This can be necessary for cases like OS and base images, where
+an application-centric Pebble service is inadequate.
+
+See the :ref:`documentation <rockcraft-yaml-entrypoint-command>` for details.
```

Comment:
```suggestion
This key defines the rock's OCI entrypoint, bypassing the default Pebble-powered 
entrypoint. It can be necessary for cases like OS and base images, where
an application-centric Pebble service is inadequate.

See the :ref:`rockcraft.yaml reference <rockcraft-yaml-entrypoint-command>` for usage 
details.
```